lightbluetent/templates/society/base.html

```diff
@@ -7,9 +7,20 @@
 
     <meta property="og:title" content="{{ _("SRCF Timeout - Virtual Freshers' Fair") }}" />
     <meta property="og:type" content="website" />
-    <!--TODO: ADD SOCIETY DESCRIPTION HERE-->
-    <meta property="og:description"
+
+    {% if society.description is not none %}
```

Comment:
Take a look at https://jinja.palletsprojects.com/en/2.11.x/templates/#truncate. This filter does everything you want intelligently, and even adds ellipsis!

Reply:
Much better!